cves/kernel/CVE-2017-17805.yml

```diff
@@ -322,10 +355,15 @@ vouch:
       * mentioning a discussion with a colleague checking the work
       * upvoting a solution on a pull request
 
+    *Note: can also include code reviews
+
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    According to the git.kernel.org site, two people signed off 
+    on the fix commit. The first person to sign off was Eric Biggers, 
```

Comment:
Love the specificity on the sign-offs for the commits!

Reply:
Thank you!